site/docs/v1/tech/samlv2/aiven.adoc

```diff
@@ -0,0 +1,121 @@
+
+.Mapping fields
+[cols="1,1"]
+|===
+|Aiven field|FusionAuth field
+
+|[field]#Metadata URL#
+|[field]#Issuer#
+
+|[field]#ACS URL#
+|[field]#Authorized redirect URLs#
+|===
+
+Scroll down to the [uielement]#Authentication response# section and change the [field]#Signing key# to the `Aiven key` you created above. Click on icon:save[role=ui-button blue,type=fas] to save your application.
+
+When redirected to the [breadcrumb]#Applications# page, view your application details by clicking on icon:search[role=ui-button green,type=fas]. In [uielement]#SAML v2 Integration details#, write down both [field]#Entity Id# and [field]#Login URL# fields.
+
+== Finish Aiven Configuration
+
+Go back to your https://console.aiven.io/[Aiven console] and click [breadcrumb]#Admin# and then on [breadcrumb]#Authentication# in the sidebar to list the authentication methods again. Click on `FusionAuth`, which will still have a `PENDING CONFIGURATION` state until you finish setting it up.
+
+Scroll down to the [uielement]#SAML config# section and click the [uielement]#Edit# button and fill in the information below.
+
+* [field]#SAML IDP Url#: the [field]#Login URL# from your FusionAuth application.
+* [field]#SAML Entity ID#: the [field]#Entity Id# from your FusionAuth application.
+* [field]#SAML Signature Algorithm#: you can leave this blank.
+* [field]#SAML Digest Algorithm#: you can leave this blank.
+* [field]#SAML Field Mapping#: you can leave this blank.
+* [field]#SAML Certificate#: paste the certificate you've copied from the `Generating certificate` two steps above.
+
+Finally, click [uielement]#Edit method# to save the changes.
+
+image::samlv2/aiven/saml-configuration.png[Configuring SAML in Aiven.,width=1200]
+
+Go to the beginning of the page and click the [field]#Enable authentication method# switch to enable it. Now scroll down to [uielement]#Signup and link accounts URLs# and  copy the [uielement]#Signup URL#. You'll use it to test the integration and then share it with others to log into your Aiven application.
+
+image::samlv2/aiven/copy-signup-url.png[Copy Signup URL from Aiven.,width=1200]
+
+== Testing
+
+Open the [uielement]#Signup URL# you copied above in an incognito tab or using another browser. You'll reach the signup page below.
+
+image::samlv2/aiven/login-sso.png[Logging in to Aiven.,width=542,role=custom]
+
+Click on [uielement]#Sign up with FusionAuth#, which will redirect you to the FusionAuth login page. Fill in your credentials and submit the form to be taken back to the Aiven console, already logged in and part of the `Developers` team.
```

Comment:
Do you need to register the user for the Aiven application?

Do you need to have any specific roles in FusionAuth that map to roles in Aiven?

Reply:
@vcampitelli did you miss this?

Reply:
No, it was late and I decided to leave it for today, that's why I didn't re-request your review yesterday :smile: I just did it. 

And no, no specific roles are needed.